run-integration-tests.sh

```diff
@@ -5,3 +5,4 @@
 ./run src/examples/zkapps/composability.ts || exit 1
 ./run src/examples/zkapps/dex/run.ts --bundle || exit 1
 ./run src/examples/zkapps/dex/happy-path-with-proofs.ts --bundle || exit 1
+./run src/examples/zkapps/dex/upgradability.ts --bundle || exit 1
```

Comment:
I had to separately run the initial DEX tests and the new tests because I was running into a weird wasm bug, will open an issue soon